Classes/Systems/Rating/RatingController.swift

```diff
@@ -57,10 +58,17 @@ final class RatingController {
     class func prompt(_ type: Prompt) {
         guard canPrompt(.system) else { return }
 
+        let typeString: String
         switch type {
-        case .system: requestSystem()
-        case .inFeed: openAppStore()
+        case .system:
+            typeString = "system"
```

Comment:
What about adding these typeStrings to the enum itself?

Reply:
Ugh that’s a way better idea. I’ll update.